mlir/test/python/dialects/sparse_tensor/dialect.py

```diff
@@ -18,10 +18,11 @@ def testEncodingAttr1D():
             "#sparse_tensor.encoding<{"
             "  map = (d0) -> (d0 : compressed),"
             "  posWidth = 16,"
-            "  crdWidth = 32"
+            "  crdWidth = 32,"
+            "  explicitVal = 1.0 : f64"
```

Comment:
just btw you can write this more easily with something like this

```python
    with Context() as ctx:
        parsed = Attribute.parse(
            textwrap.dedent("""\
              #sparse_tensor.encoding<{
                map = (d0) -> (d0 : compressed),
                posWidth = 16,
                crdWidth = 32,
                explicitVal = 1.0 : f64
            }>\
            """
            )
        )
```

Reply:
Nice! Thanks for the suggestion 🙂